action.yml

```diff
@@ -30,6 +30,9 @@ inputs:
   env:
     required: false
     description: The name of the deployment environment for Github. (Required for `start` and `deactivate-env` only)
+  envs:
```

Comment:
Can we just overload `env` to accept arrays as well, instead of introducing another configuration option?